requirements.in

```diff
@@ -38,5 +35,4 @@ html5lib==1.1
 pillow==10.2.0
 python-dateutil>=2.8.1
 jsonschema>=3.2.0
-importlib-metadata==1.7.0
```

Comment:
Strange. I added this in [this commit](https://github.com/learningequality/studio/pull/3129/commits/42bcc11cf51627023c3e3cd2453b3a71756c9185), but I didn't specify what I fixed. In the [PR](https://github.com/learningequality/studio/pull/3129), I might elude to something related to it:
> Adds jsonschema dependency and resolves cross-dependency versioning conflicts

Whatever those conflicts were, they really don't seem to be a problem now